input/sources/README.md

```diff
@@ -12,7 +12,7 @@ from the TeVCat and TeGeV cat webpages.
 
 Schema (examples from `tev-000234.yaml`):
 
-* `source_id` - Source ID in this catalog (str, e.g. `tev-000234`)
+* `source_id` - Source ID in this catalog (str, e.g. `tev-000234`). For now use tgevcat ID.
```

Comment:
Change this description to say it's an int.
And change the example ID ... if we change to tgevcat it's no longer 234.